bluesky/__init__.py

```diff
@@ -24,6 +24,10 @@
 from .plans import OuterProductAbsScanPlan
 from .plans import OuterProductDeltaScanPlan
 from .plans import Tweak
+from .plans import SpiralScan
```

Comment:
The names imported here are the *old* names, imported for back-compat. (One of the PRs I'm working on adds a comment to this effect.) I don't think we need to import the new names at top-level, but if you feel differently I have no problem with it.